jtransc-core/src/com/jtransc/plugin/reflection/MetaReflectionJTranscPlugin.kt

```diff
@@ -487,17 +487,18 @@ class MetaReflectionJTranscPlugin : JTranscPlugin() {
 
 			data class MemberInfoWithRef(val ref: Any, val mi: MemberInfo)
 
-			fun genMemberMethods(list: List<Pair<AstClass, List<MemberInfoWithRef>>>, clazz: AstClass, method: AstMethod): List<AstMethod> {
+			fun genMemberMethods(list: List<Pair<AstClass, List<MemberInfoWithRef>>>, classes: List<AstClass>, methodName: String): List<AstMethod> {
+
 				val additionalMethods: MutableList<AstMethod> = mutableListOf()
 				var methodIndex: Int = 0
 				while (methodIndex * CASES_PER_SWITCH < list.size) {
 					val mI: Int = methodIndex
 					val sI: Int = methodIndex * CASES_PER_SWITCH
-					val parentClass: AstClass = clazz
-					val parentMethod: AstMethod = method
+					val parentClass: AstClass = classes[methodIndex % classes.size]
+					val parentMethod: AstMethod? = parentClass.getMethodWithoutOverrides(methodName)
```

Comment:
As far as I remember `getMethodWithoutOverrides` return null if the method has overrides. So you can't match multiple methods with the same name with this I think